lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -292,31 +383,93 @@ class MockTimers {
     }
   }
 
-  enable(timers = SUPPORTED_TIMERS) {
+  enable(...args) {
     if (this.#isEnabled) {
-      throw new ERR_INVALID_STATE(
-        'MockTimers is already enabled!',
-      );
+      throw new ERR_INVALID_STATE('MockTimers is already enabled!');
+    }
+
+    if (args.length <= 0) {
+      this.#timersInContext = SUPPORTED_TIMERS;
+      this.#now = 0;
+      this.#toggleEnableTimers(true);
+      return;
     }
 
-    validateArray(timers, 'timers');
+    let timersDefined = false;
+    // Checks if the first argument is the timer array or the initial time
+    if (ArrayIsArray(args[0])) {
+      validateArray(args[0], 'timers');
+      // Check that the timers passed are supported
+      ArrayPrototypeForEach(args[0], (timer) => {
+        if (!ArrayPrototypeIncludes(SUPPORTED_TIMERS, timer)) {
+          throw new ERR_INVALID_ARG_VALUE(
+            'timers',
+            timer,
+            `option ${timer} is not supported`,
+          );
+        }
+      });
+
+      this.#timersInContext = args[0];
+      timersDefined = true;
+      this.#now = 0;
+    } else if (this.#isValidDateWithGetTime(args[0])) {
+      // First argument is the initial time as Date
+      this.#now = args[0].getTime();
+      this.#timersInContext = SUPPORTED_TIMERS;
+    } else if (validateNumber(args[0], 'initialTime') === undefined) {
+      // First argument is the initial time as number
+      this.#assertTimeArg(args[0]);
+      this.#now = args[0];
+      this.#timersInContext = SUPPORTED_TIMERS;
+    } else {
+      throw ERR_INVALID_ARG_VALUE(
+        'enableTimers',
+        args[0],
+        'the first argument must be an array of strings, a Date instance or a positive integer',
+      );
+    }
 
-    // Check that the timers passed are supported
-    ArrayPrototypeForEach(timers, (timer) => {
-      if (!ArrayPrototypeIncludes(SUPPORTED_TIMERS, timer)) {
-        throw new ERR_INVALID_ARG_VALUE(
-          'timers',
-          timer,
-          `option ${timer} is not supported`,
+    // Checks if the second argument is the initial time
+    if (args[1] && timersDefined) {
+      if (this.#isValidDateWithGetTime(args[1])) {
+        this.#now = args[1].getTime();
```

Comment:
same here

Reply:
Yep, same thing, if the first argument is the array, then the second argument can be either a number or a Date instance, then we call getTime on that after assuring it's a real Date